combineArrayJobOutputs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+if [ $# -le 2 ]
+then
+	echo "Usage: $0 jobID lastArrayIndex"
+	exit 1
+fi
+
+jobID=$1
+lastArrayIndex=$2
+
+# copy first file including header
+cp job_${jobID}_0/example.csv example.csv
```

Comment:
Possibly make the csv file name a parameter to the script (since it depends on the model_parameters file used)?